pntr.h

```diff
@@ -2009,6 +2169,12 @@ PNTR_API void pntr_draw_line_horizontal(pntr_image* dst, int posX, int posY, int
     }
 }
 
+PNTR_API void pntr_draw_line_horizontal_thick(pntr_image* dst, int posX, int posY, int width, int thickness, pntr_color color) {
+    pntr_draw_rectangle_fill(dst, posX, posY - thickness / 2, width, thickness, color);
+    pntr_draw_circle_fill(dst, posX, posY, thickness / 2, color);
+    pntr_draw_circle_fill(dst, posX + width, posY, thickness / 2, color);
```

Comment:
No rounded ends for thick lines:
https://www.w3schools.com/jsref/tryit.asp?filename=tryhtml5_canvas_linewidth2